2.4/blocks/aspirelists/block_aspirelists.php

```diff
@@ -8,186 +8,204 @@ function init() {
   }
 
   function get_content() {
-	global $CFG;
-	global $COURSE;
+    global $CFG;
+    global $COURSE;
 
     if ($this->content !== NULL) {
       return $this->content;
     }
 
-	$site = get_config('aspirelists', 'targetAspire');
-    $httpsAlias = get_config('aspirelists', 'targetAspireAlias');
-
-	if (empty($site))
-	{
-		$this->content->text = get_string('no_base_url_configured', 'block_aspirelists');
-		return $this->content;
-	}
-
-	$targetKG = get_config('aspirelists', 'targetKG');
-	if (empty($targetKG))
-	{
-		$targetKG = "modules"; // default to modules
-	}
-
     $hrefTarget = get_config('aspirelists', 'openNewWindow');
     $target ='_self';
     if($hrefTarget == 1){
-        $target = '_blank';
+      $target = '_blank';
```

Comment:
The rest of the code appears to use 4 spaces for tab-width.
These changes appear to be using 2 spaces. Please can we follow similar formatting to existing code?

Reply:
I've just pushed another commit to 'list-caching-dev' branch on rboyatt/aspire-moodle-integration that formats the entire file with 4 space tab-width.